.github/workflows/ci.yml

```
@@ -439,18 +439,106 @@ jobs:
+                echo "⚠️ Push failed (attempt $((PUSH_RETRY_COUNT + 1))/$MAX_PUSH_RETRIES). Checking remote..."
+                git fetch --tags --force >/dev/null 2>&1 || true
+                
+                # Check if remote already has this tag
+                if git ls-remote --tags origin "v${FINAL_VERSION}" | grep -q "v${FINAL_VERSION}"; then
+                  echo "✅ Tag v${FINAL_VERSION} already exists on remote"
+                  TAG_CREATED="true"
+                  # Clean up local tag since remote has it
+                  git tag -d "v${FINAL_VERSION}" >/dev/null 2>&1 || true
+                  break
+                fi
+                
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)